metrics_repositories.go

```diff
@@ -95,6 +103,52 @@ func (h *HarborExporter) collectRepositoriesMetric(ch chan<- prometheus.Metric)
 				ch <- prometheus.MustNewConstMetric(
 					allMetrics["repositories_tags_total"].Desc, allMetrics["repositories_tags_total"].Type, data[i].Artifact_count, data[i].Name, repoId,
 				)
+
+				repoName := strings.Split(data[i].Name, "/")
+				tagBody, _ := h.request("/projects/" + projectName + "/repositories/" + repoName[1] + "/artifacts?with_tag=true&with_scan_overview=true")
+				var scanData []map[string]interface{}
+
+				var tagData artifactsMetric
+				if err := json.Unmarshal(tagBody, &tagData); err != nil {
+					level.Error(h.logger).Log(err.Error())
+					return false
+				}
+
+				json.Unmarshal([]byte(tagBody), &scanData)
+				if len(scanData) == 0 {
+					continue
+				}
+				for key, scanData := range scanData {
+					tagName := tagData[key].Tags[0]
+
+					if len(scanData) < 16 {
+
+						continue
+					}
+					scan_overview := scanData["scan_overview"].(map[string]interface{})
+					severity := scan_overview["application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0"].(map[string]interface{})
+
+					var vurnerability float64
```

Comment:
s/vurnerability/vulnerability/

```suggestion
					var vulnerability float64
```